docs/spark/tutorials/get-started.md

```diff
@@ -197,13 +195,28 @@ This .NET app counts words with Apache Spark
 
 2. Run the following command to submit your application to run on Apache Spark:
 
-   ```powershell
-   %SPARK_HOME%\bin\spark-submit --class org.apache.spark.deploy.dotnet.DotnetRunner --master local bin\Debug\netcoreapp3.0\microsoft-spark-2.4.x-0.6.0.jar dotnet bin\Debug\netcoreapp3.0\mySparkApp.dll
+   ```console
```

Comment:
We should do the same thing as above to separate different OS versions.

Reply:
This command was the same in all of your get started articles. Should they be different for each OS?

Reply:
Please see my response above.